src/components/Constants/LoadConstants.graphql

```diff
@@ -0,0 +1,56 @@
+query LoadConstants {
+  constant {
+    activities {
+      id
+      value
+    }
+    likelyToGiveOptions {
+      id
+      value
+    }
+    locations {
+      id
+      value
+    }
+    notifications {
+      id
+      value
+    }
+    pledgeCurrencies {
```

Comment:
`pledgeCurrencies`, `pledgeFrequencies`, `preferredContactMethods` `notifications`, and  `userNotificationTitles` also have a `key` field. Idk which field to use when. They can be added later, just curious if you had any experience loading/using these.

Also apparently the stage API has different constants than production. Is there some undeployed API work? I was looking at prod when I made the Jira ticket.

Reply:
We didn't have to use them much as they were just for text.